pkg/README.md

```diff
@@ -2,12 +2,33 @@
 
 Nodejs package for x25519 key exchange and chacha20poly1305 encryption, written in Rust compiled to WASM.
 
+## Development process
```

Comment:
this was autogenerated, can delete if not needed. Assumed I should just leave it for now.